starter-dev-backend/routes/graphql/index.js

```diff
@@ -0,0 +1,19 @@
+import { graphqlHTTP } from 'express-graphql';
+import { buildSchema } from 'graphql';
+
+const schema = buildSchema(`
+  type Query {
+    hello(greeting: String!): String!
+  }
+`);
+
+const root = {
+  hello: ({ greeting }) => {
```

Comment:
Technically speaking, you should make this async.
```suggestion
  hello: async ({ greeting }) => {
```